File: src/main/java/tk/avabin/tdg/beans/DTO/SkillDto.java
package tk.avabin.tdg.beans.DTO;

import lombok.Data;
import org.springframework.context.annotation.Scope;
import org.springframework.stereotype.Component;

import java.io.Serializable;

/**
 * Created by Avabin on 18.05.2017.
 */
@Component
@Data
@Scope("prototype")
public class SkillDto implements Serializable {
    private Integer id;
    private String name;
    private Integer bonusAttributePosition;
    private Integer rank;
    private Integer otherBonuses;
}

File: src/main/kotlin/tk/avabin/tdg/beans/dtos/SkillDto.kt
package tk.avabin.tdg.beans.dtos

import org.springframework.context.annotation.Scope
import org.springframework.stereotype.Component

/**
 * Created by Avabin on 18.05.2017.
 */
@Component
@Scope("prototype")
data class SkillDto(
        var id: Int = 0,
        var name: String = "",
        var bonusAttributePosition: Int = 0,
        var rank: Int = 0,
        var otherBonuses: Int = 0
)